modules/data.remote/R/GEDI_AGB_prep.R

```diff
@@ -56,481 +92,437 @@ GEDI_AGB_prep <- function(site_info, time_points, outdir = file.path(getwd(), "G
-  site_folder <- file.path(outdir, site.id)
-  # 
-  if (file.exists(file.path(site_folder, "Error.txt"))) {
-    PEcAn.logger::logger.info("The current point is outside of GEDI domain!")
-    return(FALSE)
-  } else {
-    if (!file.exists(file.path(site_folder, "extent.txt")) | !file.exists(file.path(site_folder, "GEDI_AGB.csv"))) {
-      PEcAn.logger::logger.info("Please rerun the GEDI_AGB_prep function for this site!")
-      return(FALSE)
-    } else {
-      extent <- utils::read.table(file.path(site_folder, "extent.txt"), skip = 1) %>%
-        as.numeric %>%
-        purrr::set_names(c("ymax", "ymin", "xmin", "xmax"))
-      point.lat.lon <- matrix(c(mean(extent[c("ymin", "ymax")]), mean(extent[c("xmin", "xmax")])), nrow = 1) %>% 
-        `colnames<-`(c("lat", "lon")) %>%
-        as.data.frame
-      extent.x.y <- data.frame(matrix(c(extent["xmin"], extent["ymin"],
-                                         extent["xmax"], extent["ymin"],
-                                         extent["xmax"], extent["ymax"],
-                                         extent["xmin"], extent["ymax"]), nrow = 4, byrow = T)) %>% `colnames<-`(c("lon", "lat"))
-      res <- utils::read.csv(file.path(site_folder, "GEDI_AGB.csv"))
-      ggplot2::ggplot() +
-        ggplot2::geom_polygon(data = extent.x.y, ggplot2::aes(x = .data$lon, y = .data$lat), color="blue", fill = "white") +
-        ggplot2::geom_point(data = res, ggplot2::aes(x = .data$lon_lowestmode, y = .data$lat_lowestmode, color = .data$agbd)) +
-        ggplot2::geom_point(shape = 24, data = point.lat.lon, ggplot2::aes(x = .data$lon, y = .data$lat), size = 3) +
-        ggplot2::geom_text(data = point.lat.lon, ggplot2::aes(x = .data$lon, y = .data$lat, label=site.id, hjust=-0.1, vjust=0)) +
-        ggplot2::scale_color_distiller(palette = 'Greens', direction = 1) +
-        ggplot2::labs(color = "AGB")
-    }
-  }
-}
-#' Extract L4A GEDI above ground biomass data for the GEDI AGB prep function.
-#'
-#' @param site_info A list including site_id, longitude, and latitude.
-#' @param start_date target start date (date with YYYY-MM-DD format) for preparing GEDI AGB from remote or local database.
-#' @param end_date end date (date with YYYY-MM-DD format) for preparing GEDI AGB from remote or local database.
-#' @param outdir Directory where the final CSV file will be stored.
-#' @param nfile.min the minimum required file number to be downloaded and extracted, default is 0.
-#' @param nrow.min the minimum required observation number to be extracted, default is 0.
-#' @param buffer buffer distance (in degrees) for locate GEDI AGB searching box (default is 0.01 [~ 1 km]).
-#' @param gradient the gradient for iteratively enlarge the extent if the nfile.min or nrow.min are not reached, default is 0. If nfile.min or nrow.min is 0 this will be skipped.
+#' Detect which GEDI level 4A tiles intercept which site.
 #'
-#' @return A list of AGB data frames for each site.
+#' @title GEDI_L4A_Finder
```

Comment:
I recommend removing the explicit `@title` so that Roxygen interprets the first line above ("Detect which GEDI...") as the title -- it's a good one.
```suggestion
```